Client/Frontend/Sync/BraveCore/BraveCoreMigrator.swift

```diff
@@ -324,6 +327,10 @@ extension BraveCoreMigrator {
                 }
             }
             
+            if !didSucceed {
+                self.historyFailedMigrationItemCount = History.fetchMigrationHistory(context).count
```

Comment:
`fetchMigrationHistory` does this call return failed history items only?

Reply:
Yes it does because we are deleting very item that has been migrated but closing this PR in order to remove the pop-up directly